.travis.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+language: python
+python:
+  - "3.7"
+  - "3.8"
+install:
+  - pip install -r requirements/base.txt
+script:
+  - pytest --cov=rrpproxy
```

Comment:
Should this not be tox or something?

Reply:
Imho tox doesn't run very easily within Travis. There are plugins for it, like https://github.com/tox-dev/tox-travis. But because we are not doing very fancy stuff, just `pytest` seems fine for now.